doc/developers/advanced_installation.rst

```diff
@@ -264,11 +268,17 @@ macOS compilers from Homebrew
 Another solution is to enable OpenMP support for the clang compiler shipped
 by default on macOS.
 
-You first need to install the OpenMP library using Homebrew_::
+First install the macOS command line tools::
+
+    xcode-select --install
```

Comment:
Is it necessary when using the default clang ?

Reply:
Yes it's necessary to install homebrew. It's actually the first step of the installation instructions of homebrew but as some people might not necessarily read the homebrew installation instructions I thought it might be good to also put it here.